style.css

```diff
@@ -104,6 +104,222 @@
   list-style: none;
 }
 
+/* work-section */
+
+.work-section {
+  width: 100%;
+  height: 2788px;
+  top: 718px;
+}
+
+.work-container {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 16px, 6px;
+  border-radius: 16px;
+  margin: 36px;
+  border: 1px solid #dfe1e6;
+}
+
+.cards {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  flex-direction: column;
+  padding-left: 24px;
+}
+
+.snapshot-portofolio img {
+  width: 295px;
+  height: 220px;
+}
+
+.blocks {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 100%;
+}
+
+.txtWrapper {
+  width: 80%;
+}
+
+.project-title {
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #344563;
+  padding: 5px;
+}
+
+/* Frame 2 */
+
+.tframes {
+  display: flex;
+  gap: 10px;
+  padding: 3px 0;
+}
+
+.tframes h3 {
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.frames-list {
+  display: flex;
+  gap: 10px;
+  list-style: none;
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: rgb(122, 134, 154);
+}
+
+.frames-list img {
+  height: 10px;
+  width: 10px;
+  top: 12px;
+  border-radius: 0;
+}
+
+.project-text {
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tag-list {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  padding: 10px;
+  gap: 10px;
+}
+
+.tag-list li {
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 18px;
+  color: #6070ff;
+  background-color: rgb(235, 235, 255);
+  padding: 4px 12px;
+  border-radius: 8px;
+}
+
+.blocks-buttonWrapper {
+  padding: 5px 0;
+}
+
+.blocks-button {
+  border: 1px solid rgb(96, 112, 255);
+  border-radius: 8px;
+  padding: 12px;
+  color: #396df2;
+  background-color: #fff;
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.blocks-button:hover {
+  background-color: #6070ff;
+  color: white;
+  transition: all ease 1s;
+}
+
+.blocks-button:active {
+  background-color: hsl(235, 72%, 48%);
+  color: white;
+  transition: all ease 1s;
+}
+
+/* About Myself-section */
+.about-section {
+  width: 100%;
+  height: auto;
+  padding-bottom: 150px;
+  padding-top: 100px;
+  background-image: url(img/about-img/about\ bg.png);
+}
+
+.about-Container {
+  display: flex;
+  gap: 12px;
+  flex-direction: column;
+  width: 80%;
+  margin-left: 20px;
+}
+
+.skil-List {
+  display: flex;
+  flex-direction: column;
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  padding-top: 12px;
+  list-style-type: none;
+}
```

Comment:
- [ ] I think it would be better to add some space/margin to the right and left of the technology categories as given in the design guidelines 🤓⛏ This can be achieved by the following 👇

```
  width: 90vw;
  margin: 0 auto;
```

- [ ] Also, please note the the arrows are not exactly aligned in the design guidelines. Please consider adding some `padding-left` to arrows only ⛏

Your design | Figma design 
---| ---|
 ![image](https://user-images.githubusercontent.com/106119105/235193099-53e913ac-a1b7-4beb-aa67-1837dae50153.png) | ![image](https://user-images.githubusercontent.com/106119105/235193035-cbf5440e-6348-4fe9-8d88-87fdd4e18614.png)